configuration/persistence.md

```diff
@@ -11,59 +11,73 @@ openHAB can store data over time; this is known as persistence.
 The data may be retrieved at a later time, for example to restore your system after startup, or to prepare graphs for display on a UI.
 
 openHAB persists Item states in a database, and most popular databases are supported.
-You may have more that one persistence add-on loaded, and each of these may be configured independently.
+You may have more than one persistence add-on loaded, and each of these may be configured independently.
 
 A complete list of supported persistence add-ons may be found in the [persistence]({{base}}/addons/persistence.html) section of the on-line openHAB documentation.
 
-## Persistence Add-on Configuration
+## Add-on Configuration
 
-Each persistence add-on you install will need to be configured.
 Please refer to the specific [on-line documentation]({{base}}/addons/persistence.html) for your selected persistence add-on for configuration instructions.
```

Comment:
In most other articles we are linking the subject directly. I.e.

> Please refer to the documentation of [available persistence service add-ons]({{base}}/addons/persistence.html) for configuration instructions.

(besides I for one would write online instead of on-line...?)

Reply:
Changed to, "Please refer to the [available persistence service add-on]({{base}}/addons/persistence.html) documentation for your selected persistence add-on for configuration instructions."